trunk/NDHMS/Routing/Reservoirs/RFC_Forecasts/module_rfc_forecasts_properties.F

```diff
@@ -6,12 +6,14 @@
 ! initialized/instantiated.
 
 module module_rfc_forecasts_properties
+
     use module_reservoir, only: reservoir_properties
     implicit none
 
     ! Extend/derive rfc forecasts properties from the abstract base
     ! type for reservoir properties.
     type, extends(reservoir_properties) :: rfc_forecasts_properties_interface
+        integer :: lake_number                  ! lake number
```

Comment:
Again, perhaps just a lack of perspective, but it's not clear to me what `lake_number` is.  Is this different from `lake_id`?  A replacement?

Reply:
lake_number is lake_id. We wanted to stay consistent with the original name of 'lake_number' in the original levelpool code.

Reply:
Ok, I see.